optimum/onnxruntime/modeling_decoder.py

```diff
@@ -126,14 +135,16 @@ def __init__(
         session: onnxruntime.InferenceSession,
         config: "PretrainedConfig",
         device: torch.device,
-        use_io_binding: Optional[bool] = None,
+        use_merged: bool = False,
+        use_io_binding: bool = True,
```

Comment:
why True?

Reply:
Why did you suggested to set it as `None` in #604?

Reply:
Because if you set it as `True` by default, methods relative to IO binding are called while they should not.

Reference: https://github.com/huggingface/optimum/pull/604#issuecomment-1357361010

Moreover, setting iobinding to True by default raise the warning https://github.com/huggingface/optimum/blob/b412390c20ab856f49069cb4561fc5469eeb96e8/optimum/onnxruntime/utils.py#L222-L225 which is unhelpful.